frontend/app-development/hooks/mutations/useUpdateProcessDataTypeMutation.ts

```diff
@@ -0,0 +1,20 @@
+import { type MetaDataForm } from '@altinn/process-editor/src/contexts/BpmnConfigPanelContext';
```

Comment:
As this type is now used in two packages, process-editor and app-development, I suggest that we move it to the ```shared``` folder so we avoid importing from one package to another 😄